src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -53,6 +54,7 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
             Extensions.Guard.ArgumentNotNull(notifications, nameof(notifications));
 
             this.repositoryHost = repositoryHost;
+            activeLocalRepo = activeRepo;
```

Comment:
I tried saving just the current branch instead of the local repo, but for some reason the branch would never update in `Initialize()`